app/src/Homepage/Homepage-components.tsx

```diff
@@ -22,10 +44,17 @@ export const Sidebar: React.FC<SidebarProps> = ( props: SidebarProps ) => {
 
 
 export const LevelTab: React.FC<{ classlevel: string }> = ({ classlevel }) => {   
+    const navigate = useNavigate();
     
     const handleClick = () => {
-        console.log('Button clicked: ' + classlevel);
-        
+        if (classlevel === 'Home') {
+            // Default route
+            navigate('/' + GetDefaultRoute() + '/');
+        } else {
+            // Gets correct 'X00s' from classlevel and routes to that page
+            const levelWithoutCSE : string = classlevel.split(" ", 2)[1];
```

Comment:
Can you create a unit test that checks this logic is correct? You may need to change this logic into a function

Reply:
added